### PR TITLE
Demo app Enhancement

### PR DESCRIPTION
**Pull request by PremChavhan**

### CODE REVIEW

File: .idea/modules.xml [hunk 1]
```diff
@@ -4,6 +4,9 @@
     <modules>
       <module fileurl="file://$PROJECT_DIR$/EasyUPIPayment.iml" filepath="$PROJECT_DIR$/EasyUPIPayment.iml" />
       <module fileurl="file://$PROJECT_DIR$/EasyUpiPayment/EasyUpiPayment.iml" filepath="$PROJECT_DIR$/EasyUpiPayment/EasyUpiPayment.iml" />
+      <module fileurl="file://$PROJECT_DIR$/EasyUpiPayment-Android.iml" filepath="$PROJECT_DIR$/EasyUpiPayment-Android.iml" />
+      <module fileurl="file://$PROJECT_DIR$/EasyUpiPayment/EasyUpiPayment-Android-EasyUpiPayment.iml" filepath="$PROJECT_DIR$/EasyUpiPayment/EasyUpiPayment-Android-EasyUpiPayment.iml" />
+      <module fileurl="file://$PROJECT_DIR$/app/EasyUpiPayment-Android-app.iml" filepath="$PROJECT_DIR$/app/EasyUpiPayment-Android-app.iml" />
```
Comment (PatilShreyas) [inline .idea/modules.xml]: This files in `.idea` folder should not be included in PR.

File: EasyUpiPayment/build.gradle [hunk 2]
```diff
@@ -5,11 +5,11 @@ apply plugin: "com.jfrog.bintray"
 version = "1.1"
 
 android {
-    compileSdkVersion 29
+    compileSdkVersion 28
```
Comment (PatilShreyas) [inline EasyUpiPayment/build.gradle]: This Version should not be downgraded. Don't change build files.

File: app/src/main/java/com/example/easyupipayment/MainActivity.java [hunk 3]
```diff
@@ -29,16 +32,22 @@ protected void onCreate(Bundle savedInstanceState) {
         imageView = findViewById(R.id.imageView);
         statusView = findViewById(R.id.textView_status);
         payButton = findViewById(R.id.button_pay);
-
+        Vpa=findViewById(R.id.edittext_vpa);
+        Pname=findViewById(R.id.edittext_pname);
+        Tid=findViewById(R.id.edittext_tid);
+        Trefid=findViewById(R.id.edittext_trefid);
+        Desc=findViewById(R.id.edittext_desc);
```
Comment (PatilShreyas) [inline app/src/main/java/com/example/easyupipayment/MainActivity.java]: Coding style should be neat. Please use camel case for variables and keep spaces in assignment.

File: .idea/modules.xml (hunk 1 above)
Comment (PatilShreyas) [reply .idea/modules.xml]: These files should not be updated. Just exclude it from PR.

**Comment by PatilShreyas** [comment]
@PremChavhan can you provide Screenshot of Demo app?

File: .idea/modules.xml (hunk 1 above)
Comment (PremChavhan) [reply .idea/modules.xml]: ok i will do that changes very soon.

File: app/src/main/java/com/example/easyupipayment/MainActivity.java (hunk 3 above)
Comment (PremChavhan) [reply app/src/main/java/com/example/easyupipayment/MainActivity.java]: I will keep this suggestion in mind while doing next PR

Comment (PatilShreyas) [reply app/src/main/java/com/example/easyupipayment/MainActivity.java]: You don't have any neee to create seperate PR. Just push your local commits to `master` branch of your repo. All changes/commits will be automatically included in this PR.

**Pull request closed without merging**